s3_website.gemspec

```diff
@@ -28,7 +28,6 @@ Gem::Specification.new do |s|
   s.files         = `git ls-files`
                       .split("\n")
                       .reject { |f| f.match('sbt-launch.jar') } # Reject the SBT jar, as it is a big file
-                      .push('resources/s3_website.jar.md5') # Include the checksum file in the gem
```

Comment:
I found that when I was building the gem from source (in my fork), the installation was failing because that file doesn't actually exist in the repository.  I can definitely remove the commit that does this if you'd prefer.  Let me know and I'll get it taken care of.